vendors/zoom-demo/imagezoom.js

```diff
@@ -118,6 +118,6 @@
   }
 
   $(document).ready(function(){
-    $('[data-imagezoom]').imageZoom();
+    //$('[data-imagezoom]').imageZoom();
```

Comment:
Recommend deleting code instead of commenting it out.  Also, I think this whole function can go away rather than just its contents.